rplugin/python3/defx/column/filename.py

```diff
@@ -69,21 +55,8 @@ def syntaxes(self) -> typing.List[str]:
 
     def highlight_commands(self) -> typing.List[str]:
         commands: typing.List[str] = []
```

Comment:
Actually, I have a question here. It is necessary to clear the commands and rebuild it.
Is it better to keep the command while the column object is initialized?
If this is the case, I am willing to make these changes.

Reply:
> Is it better to keep the command while the column object is initialized?

No.  Because column syntax depends on candidates.
Have you read the column's code?

And the syntax items are cached.  So your fix is not needed.